paimon-flink/paimon-flink-cdc/src/test/java/org/apache/paimon/flink/pipeline/cdc/schema/PaimonMetadataApplierTest.java

```diff
@@ -0,0 +1,609 @@
+                        catalog.getTable(Identifier.fromString(tableId.identifier())).rowType())
+                .isEqualTo(tableSchema);
+    }
+
+    @Test
+    public void testCreateTableWithComment()
+            throws Catalog.TableNotExistException, SchemaEvolveException {
+        String databaseName = "test_" + UUID.randomUUID();
+        TableId tableId = TableId.tableId(databaseName, "test.table_with_comment");
+        CreateTableEvent createTableEvent =
+                new CreateTableEvent(
+                        tableId,
+                        org.apache.flink.cdc.common.schema.Schema.newBuilder()
+                                .physicalColumn(
+                                        "col1",
+                                        org.apache.flink.cdc.common.types.DataTypes.STRING()
+                                                .notNull(),
+                                        "comment of col1")
+                                .physicalColumn(
+                                        "col2",
+                                        org.apache.flink.cdc.common.types.DataTypes.STRING(),
+                                        "comment of col2")
+                                .physicalColumn(
+                                        "col3",
+                                        org.apache.flink.cdc.common.types.DataTypes.STRING(),
+                                        "comment of col3")
+                                .physicalColumn(
+                                        "col4",
+                                        org.apache.flink.cdc.common.types.DataTypes.STRING(),
+                                        "comment of col4")
+                                .comment("comment of table_with_comment")
+                                .build());
+        metadataApplier.applySchemaChange(createTableEvent);
+        Table table = catalog.getTable(Identifier.fromString(tableId.identifier()));
+        RowType tableSchema =
+                new RowType(
+                        Arrays.asList(
+                                new DataField(
+                                        0, "col1", DataTypes.STRING().notNull(), "comment of col1"),
+                                new DataField(1, "col2", DataTypes.STRING(), "comment of col2"),
+                                new DataField(2, "col3", DataTypes.STRING(), "comment of col3"),
+                                new DataField(3, "col4", DataTypes.STRING(), "comment of col4")));
+        Assertions.assertThat(table.rowType()).isEqualTo(tableSchema);
+        Assertions.assertThat(table.primaryKeys()).isEmpty();
+        Assertions.assertThat(table.partitionKeys()).isEmpty();
+        Assertions.assertThat(table.options()).containsEntry("bucket", "-1");
+        Assertions.assertThat(table.comment()).contains("comment of table_with_comment");
+    }
+
+    @Test
+    public void testMysqlDefaultTimestampValueConversionInAddColumn()
```

Comment:
Is "Mysql" a typo? Or maybe better to add a comment describing why this paimon test is related to mysql.

Reply:
I think the original test case is meant to cover some special default values in MySQL timestamp fields (like `CURRENT_TIMESTAMP`). Renamed it to avoid confusion.